### PR TITLE
Client: Add azimuth in alerts functions

### PR DESCRIPTION
**Pull request by florianriche**
Small PR to add the azimuth information in the client messages.

Also add default example values to the Alert Schemas for the software_hash and the azimuth.

### CODE REVIEW

File: src/app/api/schemas.py [hunk 1]
```diff
@@ -163,8 +163,8 @@ class SoftwareHash(BaseModel):
 class MyDeviceIn(Login, DefaultPosition):
     specs: str = Field(..., min_length=3, max_length=100, example="systemV0.1")
     last_ping: datetime = Field(default=None, example=datetime.utcnow())
-    angle_of_view: float = Field(..., gt=0, lt=360)
-    software_hash: str = Field(None, min_length=8, max_length=16)
+    angle_of_view: float = Field(..., gt=0, lt=360, example=10)
+    software_hash: str = Field(None, min_length=8, max_length=16, example="0123456789ABCDEF")
```
Comment (Akilditu) [inline src/app/api/schemas.py:167]: Just for me to understand, is this linked to the azimuth ?

**Review by Akilditu — DISMISSED** [review]
Thanks a lot for this PR and adding this to the client !

File: src/app/api/schemas.py (hunk 1 above)
Comment (florianriche) [reply src/app/api/schemas.py:167]: No but I took rthe opportunity.  It os not worth a Pr

File: client/pyroclient/client.py [hunk 2]
```diff
@@ -222,6 +227,7 @@ def send_alert_from_device(
         Args:
             lat: the latitude of the alert
             lon: the longitude of the alert
+            -azimuth: the azimuth of the alert
```
Comment (frgfm) [inline client/pyroclient/client.py:230]: typo here, the hyphen isn't needed

**Review by frgfm — CHANGES_REQUESTED** [review]
Thanks for the PR! Only a small typo in the docstring of the client but the rest looks good!

**Review by frgfm — APPROVED** [review]
Thanks for the edits!

**Pull request merged**